esy.json

```diff
@@ -18,13 +18,22 @@
   "devDependencies": {
     "@opam/merlin": "^3.2.2",
     "@opam/merlin-lsp": "*",
-    "ocaml": "~4.7.1000"
+    "ocaml": "~4.7.1000",
+    "@opam/httpkit": "*",
+    "@opam/httpkit-lwt": "*",
+    "@opam/logs": "*",
+    "@opam/fmt": "*",
+    "@opam/yojson": "*"
```

Comment:
I'm not opposed to `Logs` and `Fmt` as we might need them, but is there any reason you're adding `Yojson`?

Reply:
@rauanmayemir he's probably doing it for temporary testing purposes 👍

Reply:
@rauanmayemir As @wokalski said - it's temporary. I want to add only httpkit nothing else in the final version :)